tee-worker/omni-executor/intent/executors/cross-chain/src/lib.rs

```diff
@@ -1074,6 +1075,69 @@ impl<BinanceClient: BinanceApi> IntentExecutor for CrossChainIntentExecutor<Bina
 	}
 }
 
+impl<BinanceClient: BinanceApi> CrossChainIntentExecutor<BinanceClient> {
+	pub async fn create_market_order_tx(
+		&self,
+		access_token: &str,
+		order: CreateMarketOrderUnsignedTxBody,
+		account_id: &AccountId,
+	) -> Result<SendOrderTxResponse, ()> {
+		let wallet_index = order.wallet_index.clone();
+		let response = self
+			.pumpx_api
+			.create_market_order_unsigned_tx(access_token, order)
+			.await
+			.map_err(|_| log::error!("Failed to get unsigned market order tx"))?;
+
+		let unsigned_tx_string = response.data.tx_data.ok_or_else(|| ())?;
```

Comment:
Can we have an error logged here ? In other places too.

Reply:
I've logged errors